src/scanner.ts

```diff
@@ -55,15 +55,22 @@ export default class Scanner {
 
     _next() {
         if (this._s.length === 0) {
-            return false;
+          return false;
         }
-        let r = this.re.exec(this._s);
-        if (r === null) {
-            throw("cannot find next token in [" + this._s + "]");
+        // if addnotation
+        if (this._s.startsWith("<%")) {
+          let nPos = this._s.search("%>");
+          this._s = this._s.substring(nPos + 2);
```

Comment:
I think we need to add throw exception if closed template brackets don't found here
Also, i think we need add some escape mechanism
when we need use <% or %> inside queries for queries like
```
SELECT $timeSeries, '<%hostname%>' AS label, sum(gauge) FROM $table WHERE $timeFilter
```

Reply:
Correct. Will inform you when it is done.